extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -1306,14 +1342,34 @@ function constructSRARequest_(a4a, instances) {
   // TODO(bradfrizzell): Need to add support for RTC.
   dev().assert(instances && instances.length);
   const startTime = Date.now();
-  return Promise.all(
-      instances.map(instance => instance.getAdUrlDeferred.promise))
+  return Promise
+      .all(instances.map(instance => instance.getAdUrlDeferred.promise))
       .then(() => googlePageParameters(a4a, startTime))
       .then(googPageLevelParameters => {
         const blockParameters = constructSRABlockParameters(instances);
-        return truncAndTimeUrl(DOUBLECLICK_BASE_URL,
+        return truncAndTimeUrl(
```

Comment:
revert 1351 and 1352

Reply:
Hmm I do need to add an extra argument here which changes the formatting but I'm not sure I've got it right yet.